tests/unit_tests/utils/test_huggingface_hub_bucketfs__model_transfer.py

```diff
@@ -0,0 +1,71 @@
+from pathlib import Path
+from typing import Union
+from unittest.mock import create_autospec, MagicMock, call
+
+from exasol_bucketfs_utils_python.bucketfs_location import BucketFSLocation
+
+from exasol_transformers_extension.utils.bucketfs_model_uploader import BucketFSModelUploader, \
+    BucketFSModelUploaderFactory
+from exasol_transformers_extension.utils.huggingface_hub_bucketfs_model_transfer import ModelFactoryProtocol, \
+    HuggingFaceHubBucketFSModelTransfer
+from exasol_transformers_extension.utils.temporary_directory_factory import TemporaryDirectoryFactory
+from tests.utils.mock_cast import mock_cast
+
+
+class TestSetup:
```

Comment:
could this be a fixture instead?

Reply:
I use this class to group all the necessary mocks. A fixture doesn't allow me this, except if I return, in fact, a class, but then I have a class and a fixture. If we did that, we save one line in the test, but get one more parameter. So, we don't win anything, using a fixture here, because we need a new instance of the class for each test, anyway.